src/store/types/index.ts

```diff
@@ -4,12 +4,12 @@ import {
   TRemoveFromArrayAction,
   TUpdateInArrayAction,
   TPushIntoArrayAction,
-  IForm,
```

Comment:
It was a dead import, nobody was using it
Anyhow `IForm` is not an PayloadAction so it shouldn't be added here, otherwise actions would freakout
also, `IForm` already being exported here on line `19`
`export * from './form';`